main.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"os"
+
+	"fmt"
+
+	"time"
+
+	"strconv"
+
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-steplib/steps-start-android-emulator/tools"
+)
+
+// ConfigsModel ...
+type ConfigsModel struct {
+	emulatorSerial string
+	bootTimeout    int64
+	AndroidHome    string
+}
+
+// -----------------------
+// --- Functions
+// -----------------------
+
+func createConfigsModelFromEnvs() ConfigsModel {
+
+	timeout := int64(180)
+
+	if inputTimeout, err := strconv.ParseInt(os.Getenv("boot_timeout"), 10, 64); err == nil {
```

Comment:
validate the inputs in a separate function, in this method please only collect the inputs!